### PR TITLE
DOC-8925: Organize billing based on folders

### PR DESCRIPTION
**Pull request by Amruta-Ranade**
Addresses [DOC-8925](https://cockroachlabs.atlassian.net/browse/DOC-8925)

### CODE REVIEW

**Review by laurenbarker — APPROVED** [review]
All looks good to me!

File: src/current/cockroachcloud/billing-management.md [hunk 1]
```diff
@@ -61,4 +61,6 @@ Under the **Spend over time** section, you can see a table of each of your organ
 
 ## View invoices
 
-You can view all of your organization's past invoices on the **Invoices** tab of the **Billing** page. Download any invoice to see the details of your charges for a billing period.
+You can view all of your organization's past invoices on the **Invoices** tab of the **Billing** page. Download any invoice to see the details of your charges for a billing period.
```
Comment (jramos-crl) [inline src/current/cockroachcloud/billing-management.md:64]: @Amruta-Ranade one call out here - users can't actually "download" invoices. I realize this was probably already written like this but worth correcting. They can see the full invoice in the UI but they can't actually download it.

**Review by jramos-crl — CHANGES_REQUESTED** [review]
Changes look good - left one comment regarding "downloading" invoices but otherwise good to go.

File: src/current/cockroachcloud/billing-management.md (hunk 1 above)
Comment (Amruta-Ranade) [reply src/current/cockroachcloud/billing-management.md]: Fixed!

**Review by jramos-crl — APPROVED** [review]
LGTM!

File: src/current/cockroachcloud/billing-management.md [hunk 2]
```diff
@@ -61,4 +61,6 @@ Under the **Spend over time** section, you can see a table of each of your organ
 
 ## View invoices
 
-You can view all of your organization's past invoices on the **Invoices** tab of the **Billing** page. Download any invoice to see the details of your charges for a billing period.
+You can view all of your organization's past invoices on the **Invoices** tab of the **Billing** page. Click on any invoice to view the details of your charges for a billing period. 
+
+If you're using [folders]({% link cockroachcloud/folders.md %}) to organize your clusters, billing is grouped by folder. Click any folder to view invoices for individual clusters within it.
```
Comment (mdlinville) [inline src/current/cockroachcloud/billing-management.md:66]:
```suggestion
If you're using [folders]({% link cockroachcloud/folders.md %}) to organize your clusters, billing is grouped by folder. Click any folder to view invoices for individual clusters in the folder or its descendants. Within a folder, click any subfolder to view its details.
```
Is this accurate? I think it's key that the costs roll up the hierarchy to the organization root. WDYT @jramos-crl

File: src/current/cockroachcloud/folders.md [hunk 3]
```diff
@@ -200,8 +200,10 @@ To move a cluster from the organization level into a folder, or to move it from
 1. In the dialog, select the destination to move the cluster to, then click **Next**.
 1. Click **Move**.
 
+Alternatively, you can move a cluster to a new folder by browsing to the location of the destination folder, then clicking the the three-dots **Action** button and selecting **Move**.
```
Comment (mdlinville) [inline src/current/cockroachcloud/folders.md:203]:
```suggestion
To move a cluster to a new folder from the **Clusters** page:

1. Bowse to the location of the destination folder.
1. Click the the three-dots **Action** button, then select **Move**.
```
To align with the section just above. WDYT?

File: src/current/cockroachcloud/folders.md [hunk 4]
```diff
@@ -200,8 +200,10 @@ To move a cluster from the organization level into a folder, or to move it from
...
+
 {{site.data.alerts.callout_info}}
-You can also move a cluster to a new folder by browsing to the location of the destination folder, then clicking the the three-dots **Action** button and selecting **Move**.
+Clusters are [billed]({% link cockroachcloud/billing-management.md %}#view-invoices) to the folder they reside in at the end of the billing period, even if they were moved between folders during that time.
```
Comment (mdlinville) [inline src/current/cockroachcloud/folders.md:206]:
```suggestion
A cluster is [billed]({% link cockroachcloud/billing-management.md %}#view-invoices) to the folder it resides in at the end of the billing period, even if it was moved from one folder to another during the billing period.
```

File: src/current/cockroachcloud/billing-management.md (hunk 2 above)
Comment (jramos-crl) [reply src/current/cockroachcloud/billing-management.md:66]: That's accurate and yes makes it a bit clearer that it rolls up the hierarchy.

**Review by mdlinville — APPROVED** [review]

**Pull request merged**